src/features/account/components/AccountRoleTags.tsx

```diff
@@ -0,0 +1,81 @@
+import { Tag, TagLabel, TagLeftIcon, HStack } from '@chakra-ui/react';
+import { Trans } from '@lingui/macro';
+import type { FunctionComponent } from 'react';
+
+import { AuditorIcon, DeveloperIcon, ReviewerIcon } from '../../../components';
+
+export enum AccountRole {
+  DEVELOPER = 'developer',
+  AUDITOR = 'auditor',
+  REVIEWER = 'reviewer',
+}
+
+export type AccountRoleTagProps = {
+  roles: AccountRole[];
+};
+
+export const AccountRoleTags: FunctionComponent<AccountRoleTagProps> = ({
+  roles,
+}) => {
+  const roleset = new Set(roles);
+  const roleArr = Array.from(roleset);
```

Comment:
Why is this necessary?

Reply:
remove duplicate in case the components abuse by developer

Reply:
I think we can assume we won't use duplicates?

Reply:
i think so, but i will stick with that